src/node_contextify.cc

```diff
@@ -736,8 +736,10 @@ class ContextifyScript : public BaseObject {
       return;
     }
 
-    Local<String> decorated_stack = String::Concat(arrow.As<String>(),
-                                                   stack.As<String>());
+    Local<String> decorated_stack = String::Concat(
+      String::Concat(arrow.As<String>(),
+        FIXED_ONE_BYTE_STRING(env->isolate(), "\n")),
+      stack.As<String>());
```

Comment:
Just a tiny stile nit, but we prefer 4 spaces of indentation for statement continuations – that can be fixed while merging, though